docker/docker-compose.yml

```diff
@@ -38,14 +38,21 @@ services:
     restart: always
     ports:
       - 33807:33807
+    healthcheck:
+      test: ['CMD', 'mysqladmin', 'ping', '-h127.0.0.1', '-P33577']
+      interval: 5s
+      timeout: 2s
+      retries: 5
+      start_period: 25s
     environment:
       PORT: '33804' # unused in testing, but required by vttestserver
-      KEYSPACES: 'keyspace' # unused in testing, but required by vttestserver
+      KEYSPACES: 'unsharded' # unused in testing, but required by vttestserver
       NUM_SHARDS: '1' # unused in testing, but required by vttestserver
-
       MYSQL_BIND_HOST: '0.0.0.0'
       FOREIGN_KEY_MODE: 'disallow'
       VT_DIALECT: 'mysql80'
+      PLANNER_VERSION: 'gen4fallback'
+      TOPOLOGY_FLAGS: '-shutdown_grace_period 2'
```

Comment:
What are these settings?

Reply:
- `PLANNER_VERSION`: it uses the most recent planner for Vitess, allowing a fallback to the previous planner version in case the newest fails (see the [docs](https://vitess.io/docs/15.0/get-started/vttestserver-docker-image/#environment-variables))
- `TOPOLOGY_FLAGS`: added `-shutdown_grace_period 2` as suggested in [this comment](https://github.com/vitessio/vitess/issues/9936#issuecomment-1074498933), using an environment variable shown in [here](https://vitess.io/docs/13.0/reference/programs/vttablet/#managed-mysql) (which, as far as I can tell, is overridable)

Reply:
If these don't change anything (does it make the test faster or anything?), maybe we could remove them. Going with the defaults feels better, I think.

Reply:
It was the latest attempt to make Vitess faster. Removing them now, let's see whether the test suite stays within the new timeout boundaries (60 mins)